src/code/dialogs/zoneDialog.js

```diff
@@ -222,13 +179,18 @@ const ZonedrawHandler = L.Handler.extend({
       return;
     }
     L.Handler.prototype.enable.call(this);
-    // postToFirebase({ id: "analytics", action: "zonedrawStart" });
   },
 
   disable: function () {
     if (!this._enabled) return;
     L.Handler.prototype.disable.call(this);
-    // postToFirebase({ id: "analytics", action: "zonedrawEnd" });
+    this.setZIndex();
+    window.map.fire("wasabee:op:change");
+  },
+
+  setZIndex: function () {
+    window.plugin.wasabee.zoneLayerGroup.setZIndex(300);
```

Comment:
is this working ? I have doubt

Reply:
nope, but it's all I can find in Leaflet at the moment.

Reply:
what do you need ?

Reply:
to push the zones below everything but shadows and base tiles.

Reply:
try `FeatureGroup` and `bringToBack` instead of `LayerGroup`